### PR TITLE
added "Tablet-PC" and "to add"

### PR DESCRIPTION
**Pull request by NicoAiko**

### CODE REVIEW

File: ReadMe.md [hunk 1]
```diff
@@ -84,6 +85,7 @@ English | Japanese | Kana | Notes
 --------|----------|------|------
 Device, terminal | 端末 | たんまつ | Smartphones, tablets, e-readers, command-line interfaces, etc
 Japanese feature phone | ガラ携 | ガラけい | From ガラパゴス携帯電話
+Tablet-PC | タブＰＣ |  |
```
Comment (ronkorving) [inline ReadMe.md:88]: Could you add a mention of the full form タブレットＰＣ? At least from Googling this that seems to be a (more?) common way to write it.

You could make the entry as follows:

```
Tablet-PC | タブＰＣ / タブレットＰＣ | |
```

**Comment by ronkorving** [comment]
Thanks for the contribution! Just one tiny comment I hope you can fix (or respond to if I got it wrong, which is completely possible :))

**Comment by NicoAiko** [comment]
I only knew of タブPC, but as I researched, タブレットPC seems to be the "not-shortened" form of saying tablet PC, so both should be inserted :)

**Comment by ronkorving** [comment]
よろしくお願いします :)

**Comment by ronkorving** [comment]
@Zengor That's a good point. If you figure out how to best describe that (it's a tough one indeed), feel free to create a follow-up PR. In the mean time, I merged @NicoAiko's contribution.

**Pull request closed without merging**